plugin/al/usdtransaction/AL/usd/transaction/Transaction.cpp

```diff
@@ -17,13 +17,13 @@
 #include "AL/usd/transaction/TransactionManager.h"
 
 #include <iostream>
-using namespace pxr;
+PXR_NAMESPACE_USING_DIRECTIVE
 
 namespace AL {
 namespace usd {
 namespace transaction {
 
-Transaction::Transaction(const pxr::UsdStageWeakPtr& stage, const pxr::SdfLayerHandle& layer)
+Transaction::Transaction(const PXR_NS::UsdStageWeakPtr& stage, const PXR_NS::SdfLayerHandle& layer)
```

Comment:
With the PXR_NAMESPACE_USING_DIRECTIVE added above, we should be able to just drop the "PXR_NS::" off of these.

Reply:
Good catch - was just doing find/replace, should have been checking if they were needed in the first place!  Fixed now.